hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -74,11 +74,19 @@ def queries(query_group,
         salt '*' nebula.queries hour verbose=True
         salt '*' nebula.queries hour pillar_key=sec_osqueries
     '''
+
+    if isinstance(query_file, list):
+        query_file = _dict_update(query_file,
+                                  yaml.safe_load(f),
+                                  recursive_update=True,
+                                  merge_lists=True)
+
```

Comment:
Can you look over this code again? Looks like you have an uninstantiated variable (`f`), and you're trying to `_dict_update` on something that's not a dict (`query_file`, which is a list, and probably not a good final destination for the config).

You also need to do the `__salt__['cp.cache_file']()` for each file before you'll have them for reading anyway.